apps/web/app/(app)/gardens/[chain]/[garden]/[community]/page.tsx

```diff
@@ -0,0 +1,201 @@
+import { commImg, groupFlowers } from "@/assets";
+import React from "react";
+import Image from "next/image";
+import {
+  EthAddress,
+  Statistic,
+  PoolCard,
+  RegisterMember,
+  DisplayNumber,
+  IncreasePower,
+  FormLink,
+} from "@/components";
+import { initUrqlClient, queryByChain } from "@/providers/urql";
+import { Address } from "viem";
+import {
+  RegistryCommunity,
+  TokenGarden,
+  getCommunityDocument,
+  getCommunityQuery,
+} from "#/subgraph/.graphclient";
+import {
+  CurrencyDollarIcon,
+  RectangleGroupIcon,
+} from "@heroicons/react/24/outline";
+
+const { urqlClient } = initUrqlClient();
+
+export default async function CommunityPage({
+  params: { chain, garden: tokenAddr, community: communityAddr },
+}: {
+  params: { chain: number; garden: string; community: string };
+}) {
+  const { data: result, error: error } = await queryByChain<getCommunityQuery>(
+    urqlClient,
+    chain,
+    getCommunityDocument,
+    { communityAddr: communityAddr, tokenAddr: tokenAddr },
+  );
+
+  let tokenGarden = result?.tokenGarden;
+
+  let {
+    communityName,
+    members,
+    strategies,
+    communityFee,
+    registerStakeAmount,
+    registerToken,
+    protocolFee,
+    covenantIpfsHash,
+  } = result?.registryCommunity as RegistryCommunity;
+
+  const communityStakedTokens =
+    members?.reduce(
+      (acc: bigint, member) => acc + BigInt(member?.stakedTokens),
+      0n,
+    ) ?? 0;
+
+  strategies = strategies ?? [];
+
+  const signalingPools = strategies.filter(
+    (strategy) => strategy.config?.proposalType === "0" && strategy.isEnabled,
```

Comment:
Can we create a type to compare like for the pool types? For me its easy to make mistake here since we don't know what is 1 whiteout checking the types somewhere
![image](https://github.com/1Hive/gardens-v2/assets/13890618/8733b473-7ac5-4580-a8ae-2d471513ab6a)

Reply:
you are right, i will go as the image you provided